Makefile

```diff
@@ -21,7 +21,7 @@ SHELL := /bin/bash
 .DELETE_ON_ERROR:
 .SUFFIXES:
 
-SOURCES := $(shell find . -type f -name "*.go")
+SOURCES := $(shell find . -type f -name "*.go" -not -path "./bin/*" -not -path "./make/*")
```

Comment:
When vendoring Go (`make vendor-go`), this line would slow down the makefile a lot due to the number of Go files in GOROOT.